src/content/docs/licenses/license-information/usage-plans/new-relic-usage-plan.mdx

```diff
@@ -494,7 +494,9 @@ Details:
       * Pro
       * Enterprise
 
-    A New Relic organization can only be on one edition at a time. Your Order will indicate your edition.
+A New Relic organization can only be on one edition at a time. Your Order will indicate your edition.
+
+To understand what is included in each Edition, see [pricing](https://newrelic.com/pricing).
```

Comment:
A couple of things:

I'm not really sure that the wording of "can be on one edition at a time" is really the best. I could just be being curmudgeonly about it, but it seems to read weirdly to me. What do you think of something like this? "Your order indicates what edition your organization uses, and it can only use one edition at a time."? 

Should 'Order' and 'Edition' be capitalized?